### PR TITLE
[ZEPL-3858] add docs Logging Console

### PR DESCRIPTION
**Pull request by torreswoo**
# What is this PR?
add docs Logging console
- [x] How to use logging console. & demo (.gif)
- [x] explain log messages

image
image

### CODE REVIEW

**Comment by torreswoo** [comment]
@sejunra I didnt write documentation much. i added some explanation about the log message. need some idea and comment. please help this.

File: docs/guide/logging_console.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Logging Console in ZEPL
+
+A Logging Console that user can click from the notebook to show interpreter log and error output.
+
+ 
+## How to use Logging Console
+
+ZEPL supports logging console for Python, Spark and JDBC interpreter.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: "... and JDBC interpreters." (plural)

File: docs/guide/logging_console.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+To begin using the Logging Console, simply the run the paragraph you're working on.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Remove "the" - "simply run the..."

File: docs/guide/logging_console.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+If you shutdown the notebook, Logging Console is automatically closed.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: change "is automatically" to "will be automatically"

File: docs/guide/logging_console.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+After click `console` panel, start to get lastest logs from interpreter.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change to "The Log Console is located at the bottom of the notebook. To open, click on the `Console` tab.

File: docs/guide/logging_console.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Logging Console in ZEPL
+
+A Logging Console that user can click from the notebook to show interpreter log and error output.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change the whole sentence to "ZEPL provides a Log Console for a given notebook that the user can open to view the interpreter logs. The output stream in the Log Console will be specific to the given notebook and its interpreters.

File: docs/guide/logging_console.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+ 
+## How to use Logging Console
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change to "How to use the Log Console"

File: docs/guide/logging_console.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+ZEPL supports logging console for Python, Spark and JDBC interpreter.
+To begin using the Logging Console, simply the run the paragraph you're working on.
+If you shutdown the notebook, Logging Console is automatically closed.
+After click `console` panel, start to get lastest logs from interpreter.
+
+<br>
+Checking Highlight `ERROR` log. you can get some cue. Also you are able to see processing log.
+  
+<img src="../../img/logging_console_demo.gif" class="image-box big-img" />
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Image should be replaced w/ one that shows the open log console with ideally a red error line as well.

File: docs/guide/logging_console.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+ 
+## How to use Logging Console
+
+ZEPL supports logging console for Python, Spark and JDBC interpreter.
+To begin using the Logging Console, simply the run the paragraph you're working on.
+If you shutdown the notebook, Logging Console is automatically closed.
+After click `console` panel, start to get lastest logs from interpreter.
+
+<br>
+Checking Highlight `ERROR` log. you can get some cue. Also you are able to see processing log.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Let's remove this sentence.

File: docs/guide/logging_console.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  
+<img src="../../img/logging_console_demo.gif" class="image-box big-img" />     
+
```
Comment (sejunra) [inline docs/guide/logging_console.md]: ERROR logs are highlighted in red.

File: docs/guide/logging_console.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+To begin using the Logging Console, simply the run the paragraph you're working on.
+If you shutdown the notebook, Logging Console is automatically closed.
+After click `console` panel, start to get lastest logs from interpreter.
+
+<br>
+Checking Highlight `ERROR` log. you can get some cue. Also you are able to see processing log.
+  
+<img src="../../img/logging_console_demo.gif" class="image-box big-img" />     
+ 
+* Notice : Logging Console is only activated in running notebook.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change to "Note: You will only see log lines in the Log Console if the notebook is running."

File: docs/guide/logging_console.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+Checking Highlight `ERROR` log. you can get some cue. Also you are able to see processing log.
+  
+<img src="../../img/logging_console_demo.gif" class="image-box big-img" />     
+ 
+* Notice : Logging Console is only activated in running notebook. 
+
+<br>
+
+## About the log in Logging Console 
+The interpreter logs is kind of [Apache Zeppelin Interpreter](https://zeppelin.apache.org/docs/0.8.0/usage/interpreter/overview.html) log.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change to "The interpreter logs are based on the [Apache ..."

File: docs/guide/logging_console.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+- Timestamp : [ISO8601](https://en.wikipedia.org/wiki/ISO_8601)
+- Log level : `INFO`, `ERROR`, `WARN`
+- Thread : the name of the thread that generated the logging event.
+- message : the application supplied message associated with the logging event.
+
+<br>
+
+This is sample Spark Interpreter log in Logging Console.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change to "This is a sample Spark Interpreter log in the Log Console."

File: docs/guide/logging_console.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
+# Logging Console in ZEPL
```
Comment (sejunra) [inline docs/guide/logging_console.md]: Change name to "Log Console"

**Review by sejunra — CHANGES_REQUESTED** [review]
grammar changes and let's keep the name of it to "Log Console". That's what I thought it was.

**Comment by torreswoo** [comment]
@sejunra Thanks for your comments. i resolve these. help review it.

File: docs/guide/logging_console.md [hunk 14]
```diff
@@ -1,26 +1,24 @@
...
  
-## How to use Logging Console
+## How to use the Log Console
 
-ZEPL supports logging console for Python, Spark and JDBC interpreter.
-To begin using the Logging Console, simply the run the paragraph you're working on.
-If you shutdown the notebook, Logging Console is automatically closed.
-After click `console` panel, start to get lastest logs from interpreter.
+ZEPL supports logging console for Python, Spark and JDBC interpreters.
+To begin using the Logging Console, simply run the paragraph you're working on.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: change "Logging Console" to "Log Console"

File: docs/guide/logging_console.md [hunk 15]
```diff
@@ -1,26 +1,24 @@
...
+If you shutdown the notebook, Logging Console will be automatically closed.
```
Comment (sejunra) [inline docs/guide/logging_console.md]: change "Logging Console" to "Log Console"

**Review by sejunra — CHANGES_REQUESTED** [review]
a couple of mentions of "Logging Console". Please change to "Log Console". And please change the image that shows the whole log console in line 14. Thx.

**Comment by torreswoo** [comment]
@sejunra Thanks i resolve it again.

**Review by sravan-s — COMMENTED** [review]
Shouldn't we mention that log console isn't available for normal customers?

**Comment by torreswoo** [comment]
@sravan-s i move the log console category into enterprise section.

**Review by sejunra — APPROVED** [review]

**Pull request merged**